templates/core/sitemapindex.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    {% for which in sitemaps -%}
```

Comment:
@Kixiron do you know if tera happens to have a "character range" operator? I found `range(end=5)` but I'm not sure if it works for characters instead.